three-node-cluster/3-node-cluster.yml

```diff
@@ -0,0 +1,659 @@
+        'Fn::Join':
+          - |+
+
+          - - '#!/bin/bash'
+            - 'export PATH=~/.local/bin:$PATH'
+            - apt-get update
+            - DEBIAN_FRONTEND=noninteractive apt-get install -yq nginx
+            - mkdir -p /etc/nginx/ssl/is
+
+  WSO2ISNodeLaunchConfigurationsuffix:
+    Type: 'AWS::AutoScaling::LaunchConfiguration'
+    Properties:
+      # Uncomment below to assign a public IP to the IS instance.
+      # AssociatePublicIpAddress: true
+      ImageId: !FindInMap
+        - WSO2ISAMIRegionMapsuffix
+        - !Ref 'AWS::Region'
+        - AMI
+      InstanceType: !Ref WSO2InstanceType
+      BlockDeviceMappings:
+        - DeviceName: /dev/sda1
+          Ebs:
+            VolumeSize: '20'
+            VolumeType: gp2
+            DeleteOnTermination: 'true'
+      KeyName: !Ref KeyPairName
+      SecurityGroups:
+        - !Ref WSO2ISSecurityGroupsuffix
+      UserData: !Base64
+        'Fn::Join':
+          - |+
+
+          - - '#!/bin/bash'
+            - 'export PATH=~/.local/bin:$PATH'
+            - apt-get update
+            - DEBIAN_FRONTEND=noninteractive apt-get install -yq mysql-client
+            - DEBIAN_FRONTEND=noninteractive apt-get install -y unzip
+            - curl -o jdk-setup.tar.gz https://performance-is-packs.s3.amazonaws.com/resources/jdk-11.0.15.1_linux-x64_bin.tar.gz
+            - sudo mkdir /usr/lib/jvm
+            - sudo tar -xvf jdk-setup.tar.gz -C /usr/lib/jvm
+            - sudo mv /usr/lib/jvm/jdk* /usr/lib/jvm/jdk
+            - sudo update-alternatives --install "/usr/bin/java" "java" "/usr/lib/jvm/jdk/bin/java" 1
+            - sudo update-alternatives --install "/usr/bin/javac" "javac" "/usr/lib/jvm/jdk/bin/javac" 1
+            - sudo update-alternatives --install "/usr/bin/javaws" "javaws" "/usr/lib/jvm/jdk/bin/javaws" 1
+            - sudo chmod a+x /usr/bin/java
+            - sudo chmod a+x /usr/bin/javac
+            - sudo chmod a+x /usr/bin/javaws
+            - export JAVA_HOME=/usr/lib/jvm/jdk
+            - sudo su
+            - echo "export JAVA_HOME=/usr/lib/jvm/jdk" >>/etc/environment
+            - source /etc/environment
+            - exit
+    DependsOn:
+      - WSO2ISSecurityGroupsuffix
+      - WSO2ISVPCInternetGatewayAttachmentsuffix
+      - WSO2ISDBInstancesuffix
+
+  WSO2ISNode1AutoScalingGroupsuffix:
+    Type: 'AWS::AutoScaling::AutoScalingGroup'
+    Properties:
+      LaunchConfigurationName: !Ref WSO2ISNodeLaunchConfigurationsuffix
+      DesiredCapacity: 1
+      MinSize: 1
+      MaxSize: 1
+      VPCZoneIdentifier:
+        # Use public subnet to assign a public IP to the IS instance.
+        # - !Ref WSO2ISPublicSubnet1suffix
+        - !Ref WSO2ISPrivateSubnet1suffix
+      Tags:
+        - Key: Name
+          Value: suffix_WSO2ISInstance_1
+          PropagateAtLaunch: 'true'
+        - Key: cluster
+          Value: is
+          PropagateAtLaunch: 'true'
+        - Key: User
+          Value: 'sachinm@wso2.com'
```

Comment:
This should be parameterized not hard coded

Reply:
Removed by https://github.com/wso2/performance-is/pull/128/commits/ef88a709e62a267d93e2cf768f66851ed0a8035e